js/ui/appDisplay.js

```diff
@@ -1355,7 +1355,6 @@ var AppDisplay = class AppDisplay {
 
     selectApp(id) {
         this._showView(Views.ALL);
-        this._views[Views.ALL].view.selectApp(id);
```

Comment:
This line should not be removed, it should be changed to `this._allView.selectApp(id);`

Reply:
Updated